src/main/java/software/amazon/cloudformation/AbstractWrapper.java

```diff
@@ -383,6 +385,12 @@ protected void writeResponse(final OutputStream outputStream, final ProgressEven
         }
 
         String output = this.serializer.serialize(response);
+        if (response.getStatus() != OperationStatus.IN_PROGRESS) {
+            // if status is not IN_PROGRESS, it means the response has been sanitized
+            final String logicalResourceId = getLogicalResourceId(request);
+            final String stackId = getStackId(request);
+            this.log(String.format("StackId=%s LogicalResourceId=%s Response=%s", stackId, logicalResourceId, output));
```

Comment:
should we log whole request?

Reply:
Request is not sanitized. We potentially can add the request as well after sanitized the request (e.g. sanitized the model and also other metadata). 

But I think we need to be a little bit more careful to add it step by step.  Besides, we should add the request log at the beginning of the handler. 

I would prefer to skip the log request for this PR and we can add it further.